### DISCUSSION TITLE
Welcome to GC4_Localization Discussions!

### DISCUSSION THREAD

**Discussion by Draginol** [Draginol/GC4_Localization#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kevinchen320** [comment]
Hi I'm Kevin, a Chinese player who happens to know a bit English.
Your AI-based Chinese translation sucks know, but that's exactly why I want to help.
However, I have some trouble doing this, as well as some ideas I want to share with you DEVs. Also, I just need to know more in order to start my work swiftly.
I'll say these in 3 categories: Green light for things that I can solve but just want to let you know, Yellow for something that I really aren't confortable with, and Red for those issues that makes me inable to start my work unless they're solved (at least partially solved...)

Here are my thoughts:

Green light:
- **Tools are too confusing for users without programming backgrounds.** I just know some English but not Python. I have some basic knowledge so I can use that GUI APP you guys created(and even figured out how to save my changes, how smart lol), but when I tried to pyinstaller it into a .exe file, I failed. I mean I can work with this tool and save my changes but, improving it so that more people in the community can work without worrying about 'how to do it' would be better not just for me, but for all the warm-hearted people from different parts of the world and attract more people to join.
- **Open AI is unavailable for Chinese users.** Not a big problem, for the directory comes with AI translated sentences. Just to let you guys know thats all.
- **GitHub is not *THAT* accessible.** Chinese users need to use VPN to smoothly use GitHub. I mean, yes, we can access it, yet the bigger problem is, I don't think all people around the world ust GitHub that often, if they don't have programming backgrounds.

Yellow light:
- **Need a list of some sort of 'Keyword Dictionary' for in-game words.** I don't mean you guys need to tell me (or other translators for different language) what, for example, a 'constructor' should be called in Chinese (or other target languages), but having a list of Keywords in the game would significantly benifit us translators. It would be much less confusing for translators to determine how to translate those frequently occurring words.
- **Having a general Guidebook would be very helpful.** Doesn't need to be *THAT* detailed, but I assume knowing how to deal with things like [ICON=RaceType_CarbonBased_Icon] (I guess I can just ignore it, right?) would reduce confusion in work and avoid potential rework as much as possible.

Red light:
- (I think) **WE NEED A PROPER PROGRESS-CHEKING TOOL TO COORDINATE!** As far as i think, no community translator would be happy when they find themselves have just done some work which has already been done by others! I strongly think we need such a platform for coordination. Do you guys have one right now? Would be good to have one just like the borrowing systems used in libraries where we can 'claim' part of the work and 'keep' it for a period of time to work on it knowing that nobody is doing the same work, and return the edited file or files to the directory. I think it would be easily done on GitHub, so I strongly recommend you DEVs to set the rules guiding the coordination/cooperation. for if so, not only we translators can work more effortlessly but also makes you guys easy to check on the progress or post new information/requirements if necessary.

That's all for now, and I really hope you DEVs can think about my opinions. I'd be glad if I get response form you guys, whether in GitHub or (as I said before, GitHub is not *THAT* convenient for Chinese users) more preferably, by Email. I don't think you guys use Chinese real-time communication softwares so here's my Email address: 1815883481@qq.com. I'm also looking forward to share suggestions on gameplay. I don't know how much I can help but, I'll be waiting your replies and I hope I can help.

**Reply by kevinchen320** [reply]
I'll put this page into my bookmark anyway...
Though I doubt if I'll be frequently checking this page due to I cant access GitHub without a VPN...
It's annoying to use VPN... Yes it's just a little click for me, but I don't really use it unless I *REALLY* need to see something...
Just like (maybe) a programmer won't like to think about codes on a vacation... Not a perfect metaphor though.
